alacritty_terminal/src/term/mod.rs

```diff
@@ -436,23 +436,24 @@ pub mod mode {
 
     bitflags! {
         pub struct TermMode: u16 {
-            const SHOW_CURSOR         = 0b000_0000_0000_0001;
-            const APP_CURSOR          = 0b000_0000_0000_0010;
-            const APP_KEYPAD          = 0b000_0000_0000_0100;
-            const MOUSE_REPORT_CLICK  = 0b000_0000_0000_1000;
-            const BRACKETED_PASTE     = 0b000_0000_0001_0000;
-            const SGR_MOUSE           = 0b000_0000_0010_0000;
-            const MOUSE_MOTION        = 0b000_0000_0100_0000;
-            const LINE_WRAP           = 0b000_0000_1000_0000;
-            const LINE_FEED_NEW_LINE  = 0b000_0001_0000_0000;
-            const ORIGIN              = 0b000_0010_0000_0000;
-            const INSERT              = 0b000_0100_0000_0000;
-            const FOCUS_IN_OUT        = 0b000_1000_0000_0000;
-            const ALT_SCREEN          = 0b001_0000_0000_0000;
-            const MOUSE_DRAG          = 0b010_0000_0000_0000;
-            const MOUSE_MODE          = 0b010_0000_0100_1000;
-            const ALTERNATE_SCROLL    = 0b100_0000_0000_0000;
-            const ANY                 = 0b111_1111_1111_1111;
+            const SHOW_CURSOR         = 0b0000_0000_0000_0001;
+            const APP_CURSOR          = 0b0000_0000_0000_0010;
+            const APP_KEYPAD          = 0b0000_0000_0000_0100;
+            const MOUSE_REPORT_CLICK  = 0b0000_0000_0000_1000;
+            const BRACKETED_PASTE     = 0b0000_0000_0001_0000;
+            const SGR_MOUSE           = 0b0000_0000_0010_0000;
+            const MOUSE_MOTION        = 0b0000_0000_0100_0000;
+            const LINE_WRAP           = 0b0000_0000_1000_0000;
+            const LINE_FEED_NEW_LINE  = 0b0000_0001_0000_0000;
+            const ORIGIN              = 0b0000_0010_0000_0000;
+            const INSERT              = 0b0000_0100_0000_0000;
+            const FOCUS_IN_OUT        = 0b0000_1000_0000_0000;
+            const ALT_SCREEN          = 0b0001_0000_0000_0000;
+            const MOUSE_DRAG          = 0b0010_0000_0000_0000;
+            const MOUSE_MODE          = 0b0010_0000_0100_1000;
+            const UTF8_MOUSE          = 0b0100_0000_0000_0000;
```

Comment:
`MOUSE_MODE` should include `UTF8_MOUSE`.

Reply:
It shouldn't, it's the same thing as `MOUSE_SGR`, which is not a part of MOUSE_MODE, UTF8_MOUSE is an optional encoding here. Am I missing something?

Reply:
Oh, I thought that SGR was included too, completely missed that. Good point.